packages/effect/src/Layer.ts

```diff
@@ -31,6 +31,7 @@ import * as defaultServices from "./internal/defaultServices.js"
 import * as fiberRuntime from "./internal/fiberRuntime.js"
 import * as internal from "./internal/layer.js"
```

Comment:
Didn't update this from

```ts
import * as internal from "./internal/layer.js"
```
to
```ts
import * as internalLayer from "./internal/layer.js"
```

just because it made the diff so much larger, but I could certainly do that for consistency if thats desired